### PR TITLE
Added community and developers subdomains, improved naming.

### PR DESCRIPTION
**Pull request by marcoslater**

### CODE REVIEW

File: src/chrome/content/rules/getmondo.co.uk.xml [hunk 1]
```diff
@@ -1,7 +1,9 @@
-<ruleset name="get Mondo.co.uk">
+<ruleset name="Get Mondo">
 
 	<target host="getmondo.co.uk" />
 	<target host="www.getmondo.co.uk" />
+	<target host="community.getmondo.co.uk" />
+	<target host="developers.getmondo.co.uk" />
```
Comment (J0WI) [inline src/chrome/content/rules/getmondo.co.uk.xml]: Could you also add devslack.getmondo.co.uk?

**Comment by J0WI** [comment]
Note that serves preloaded HSTS header, but is not yet preloaded.

**Comment by marcoslater** [comment]
@J0WI Good spot, added. 

I've also noticed they serve HSTS preload, but aren't on the list. Perhaps forgot?

**Comment by J0WI** [comment]
They are still pending, so I'll merge this now.
We may remove the rule in future when they are on the list for sure.

**Pull request merged**